t/unit/test_pidbox.py

```diff
@@ -58,6 +60,26 @@ def test_publish_reply_ignores_InconsistencyError(self):
             )
             producer.publish.assert_called()
 
+    def test_publish_reply_handles_redis_OperationalError_wth_no_route_error_msg(self):
+        mailbox = pidbox.Mailbox('reply.celery')(self.connection)
+        exchange = mailbox.reply_exchange.name
+        channel = self.connection.channel()
+        mailbox.reply_queue(channel).declare()
+        ticket = uuid()
+        # Using Channel._lookup as a proxy (in absence of a redis integration test) to mock the actual
+        # redis.get_table call which produces the OperationalError, since _lookup is just a level above
+        # redis.get_table on the stack trace.
+        with patch.object(kombu.transport.virtual.Channel, '_lookup') as simulate_redis_get_table_err:
+            # raise the actual redis error
+            simulate_redis_get_table_err.side_effect = OperationalError(
+                redis.NO_ROUTE_ERROR.format(exchange, mailbox.oid))
+            try:
```

Comment:
Please use `pytest.raises()`.

Reply:
We're trying to assert that the particular Error is NOT raised now, so can't use `pytest.raises()`.